docs/modules/data-structures/pages/cpmap.adoc

```diff
@@ -7,7 +7,8 @@ distributed key-value implementation, these operations involve remote calls and
 performance differs from non-distributed key-value data structures; for example, `java.util.HashMap`. 
 `CPMap` is only available in the Enterprise Edition.
 
-NOTE: Your license must include `ADVANCED_CP` to activate this feature.
+NOTE: Your license must include `ADVANCED_CP` to activate this feature and you must use the
+Enterprise Edition client.
```

Comment:
Two questions:

1. Can you include ADVANCED_CP in an OS license?
2. If you must use the EE client, can you also be using OS clients that do not require ADVANCED_CP; and is it likely that anyone with an EE license is using OS clients?

If no to both, I'd suggest we can shorten to:

NOTE: Your Enterprise Edition license must include `ADVANCED_CP` to activate this feature.

or

NOTE: You must have an Enterprise Edition license with `ADVANCED_CP` [enabled] to use this feature.

Reply:
Good enough for me, thanks ;-)